packages/plugin-ext/src/plugin/type-converters.ts

```diff
@@ -169,9 +169,9 @@ export function fromRangeOrRangeWithMessage(ranges: theia.Range[] | theia.Decora
         });
     } else {
         return ranges.map((r): DecorationOptions =>
```

Comment:
Nitpick: Formatting looks off as well here

```suggestion
        return ranges.map(r => ({ range: fromRange(r) }));
```

Reply:
I have made the appropriate fix.